sites/friday/src/get-started/here/make-changes.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Make changes to your project
+sidebarTitle: Revisions
+weight: 50
+description: How to work on a daily basis with an {{% vendor/name %}} project?
+---
+
+Now that your project is deployed, you can start making changes to it.
+For example, you might want to fix a bug or add a new feature.
+It's this component of Upsun - preview environments - that you'll come to recognize as a key feature.
+
+In your project, the `main` branch (or `master` if you change it during project creation) always represents the production environment.
+Other branches are for developing new features, fixing bugs, or updating the infrastructure.
+
+To make changes to your project, follow these steps:
+
+1. Create a new environment (a Git branch) to make changes without impacting production:
+{{< codetabs >}}
++++
+title={{% vendor/name %}} Git repository
++++
+   ```bash {location="Terminal"}
+   {{% vendor/cli %}} branch feat-a
+   ```
+   This command creates a new local `feat-a` Git branch based on the `main` Git branch
+   and activates a related environment on {{< vendor/name >}}.
+   The new environment inherits the data (service data and assets) of its parent environment (the production environment here).
+
+   {{< note title="Warnings" >}}
+   **First**, you will be charged by [activated environments](/environments/deactivate-environment.md#reactivate-an-environment).
+   So, if you would like to create an inactive environment, waiting for you to be ready to test, you could use the following command:
+   ```bash {location="Terminal"}
+   git branch checkout -b feat-a && git push -u upsun feat-a
+   ```
+
+   **Secondly**, if you have set up an integration and have enabled `prune_branches` (true by default), then `upsun branch` command will fail at the end of the deployment because the integration will sync with the source and delete the new branch since it doesn’t exist with the canonical source.
```

Comment:
shouldn't this warning be on the "using third party provider" tab? If the user has a local repo and isn't using an integration, they wont have this issue.